packages/core-components/src/layout/Page/Page.tsx

```diff
@@ -29,7 +29,7 @@ const useStyles = makeStyles<BackstageTheme>(
       gridTemplateRows: 'max-content auto 1fr',
       gridTemplateColumns: 'auto 1fr auto',
       height: '100%',
-      overflowY: 'auto',
+      // overflowY: 'auto',
```

Comment:
this other approach tries to find the scrollable element automatically. The downside is that if we find a node with overflow enabled but not scrollable due to its size (like here ☝️) this approach might not work